### PR TITLE
[FSSDK-9107] Update changelog and version for advanced audience targeting

### PR DESCRIPTION
**Pull request by Mat001**
Summary
-------

-  Update changelog and version for advanced audience targeting

Issues
------

-  [FSSDK-9107](https://jira.sso.episerver.net/browse/FSSDK-9107)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,40 @@
...
+* You can use ODP, a high-performance [Customer Data Platform (CDP)]( https://www.optimizely.com/optimization-glossary/customer-data-platform/), to easily create complex real-time segments (RTS) using first-party and 50+ third-party data sources out of the box. You can create custom schemas that support the user attributes important for your business, and stitch together user behavior done on different devices to better understand and target your customers for personalized user experiences. ODP can be used as a single source of truth for these segments in any Optimizely or 3rd party tool.
+* With ODP accounts integrated into Optimizely projects, you can build audiences using segments pre-defined in ODP. The SDK will fetch the segments for given users and make decisions using the segments. For access to ODP audience targeting in your Feature Experimentation account, please contact your Optimizely Customer Success Manager.
```
Comment (jaeopt) [inline CHANGELOG.md:10]: Let's remove these bullets at the top level to be consistent with other SDKs (see https://github.com/optimizely/swift-sdk/blob/d5e190b0aacc423c98ea133eb6d386eeb2bb019e/CHANGELOG.md)

**Review by jaeopt — CHANGES_REQUESTED** [review]
A couple of changes needed. Looks good other than those.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,40 @@
...
+* When an `OptimizelyUserContext` is created, the SDK will automatically send an identify request to the ODP server to facilitate observing user activities. 
+
+New APIs added to `OptimizelyClient`: 
+
+* `sendOdpEvent()`: customers can build/send arbitrary ODP events that will bind user identifiers and data to user profiles in ODP.
+
+For details, refer to our documentation pages:  
+
+* [Advanced Audience Targeting](https://docs.developers.optimizely.com/feature-experimentation/docs/optimizely-data-platform-advanced-audience-targeting)  
+* [Server SDK Support](https://docs.developers.optimizely.com/feature-experimentation/v1.0/docs/advanced-audience-targeting-for-server-side-sdks)
```
Comment (andrewleap-optimizely) [inline CHANGELOG.md:26]:
```suggestion
* [Server SDK Support](https://docs.developers.optimizely.com/feature-experimentation/docs/advanced-audience-targeting-for-server-side-sdks) 
```

File: optimizely/version.py [hunk 3]
```diff
@@ -11,5 +11,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-version_info = (4, 1, 1)
-__version__ = '.'.join(str(v) for v in version_info)
+version_info = (5, 0, 0)
+__version__ = '.'.join(str(v) for v in version_info) + '-beta'
```
Comment (andrewleap-optimizely) [inline optimizely/version.py:15]:
```suggestion
version_info = (5, 0, '0-beta')
__version__ = '.'.join(str(v) for v in version_info)
```

**Review by andrewleap-optimizely — CHANGES_REQUESTED** [review]
Just a couple small suggestions

**Review by andrewleap-optimizely — APPROVED** [review]
LGTM

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**